preprocessors/ner/ner.py

```diff
@@ -0,0 +1,257 @@
+# flake8: noqa
```

Comment:
Why did you add `noqa` to this file? If I understand correctly, you wrote all of this?